MyWeight/Extensions/UITableViewCell+ReuseIdentifier.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  UITableViewCell+ReuseIdentifier.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 22/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import UIKit
+
+extension UITableViewCell {
+    /**
+     - returns: a UINib with the default Nib name (matches the class name).
+        If the nib file is not found, this will cause a exception crash.
+     */
+    static func nib() -> UINib {
+        return UINib(nibName: defaultNibName, bundle: nil)
+    }
+    
+    /**
+     Default nib name for this class (matches the class name).
+     */
+    static var defaultNibName: String {
+        return String(self)
+    }
+    
+    /**
+     Default reuse identifier for this class (matches the class name).
+     */
+    static var defaultReuseIdentifier: String {
+        return String(self)
```

Comment:
I have no idea. I first saw this on Natasha 🤖's blog: https://www.natashatherobot.com/nsstringfromclass-in-swift/